ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productcart/productcart.html

```diff
@@ -0,0 +1,5 @@
+<div class ='product'>
+      <div id="mydiv" data-custom-property="${properties.productcart}" /> 
```

Comment:
@nitheshkumar-hcl : why is id mydiv ? what does this do ?

Reply:
It is used for reading the dialog value from input